container-images/js-and-gpu/Dockerfile

```diff
@@ -1,30 +1,10 @@
-FROM graphistry/gpu-base:8.0
-RUN set -ex \
-  && for key in \
-    9554F04D7259F04124DE6B476D5A82AC7E37093B \
-    94AE36675C464D64BAFA68DD7434390BDBE9B9C5 \
-    0034A06D9D9B0064CE8ADF6BF1747F4AD2306D93 \
-    FD3A5288F042B6850C66B31F09FE44734EB7990E \
-    71DCFD284A79C3B38668286BC97EC7A07EDE3FC1 \
-    DD8F2338BAE7501E3DD5AC78C273792F7D83545D \
-    B9AE9905FFD7803F25714661B63B535A4C206CA9 \
-    C4F0DFFF4E8C1A8236409D08E73BC641CC11F4C8 \
-  ; do \
-    gpg --keyserver ha.pool.sks-keyservers.net --recv-keys "$key"; \
-  done
+FROM graphistry/gpu-base:9.1
 
-ENV NPM_CONFIG_LOGLEVEL info
-ENV NODE_VERSION 9.6.0
+RUN apt-get update && apt-get install -y \
+        build-essential \
+        curl && \
+    curl -sL http://deb.nodesource.com/setup_9.x | bash && \
```

Comment:
We're curling something fetched not via TLS and piping it into bash?

Reply:
This is based on how the Node dockerfile is created btw https://github.com/nodejs/docker-node/blob/master/9/Dockerfile

Reply:
nodesource hosts on https too: https://deb.nodesource.com/setup_9.x

Reply:
also 10: https://deb.nodesource.com/setup_10.x